src/BusinessLayer/SerialReporting.cpp

```diff
@@ -353,7 +353,8 @@ void SerialReporting::sendLights()
     const unsigned int unframedPacketLength = LIGHTS_LENGTH + CHECKSUM_LENGTH;
     unsigned char packetPayload[unframedPacketLength];
     packetPayload[0] = CcsDefines::LIGHTS_PKG_ID;
-    packetPayLoad[1] = CcsDefines::LIGHTS_ALIVE;
+    bool lightsAliveArray[] = {lightsData_.alive};
```

Comment:
You'll have to edit the actual `lightsData.h` file to get it to work. `alive` hasn't been defined yet.